Source/WebCore/platform/graphics/gstreamer/GStreamerRegistryScanner.cpp

```diff
@@ -416,6 +415,12 @@ void GStreamerRegistryScanner::initializeDecoders(const GStreamerRegistryScanner
         { ElementFactories::Type::Demuxer, "video/quicktime, variant=(string)3gpp", { "video/3gpp"_s }, { } },
         { ElementFactories::Type::Demuxer, "video/x-ms-asf", { }, { } },
     };
+
+    if (const char* hlsSupport = getenv("WEBKIT_GST_ENABLE_HLS_SUPPORT")) {
+        if (!strcmp(hlsSupport, "1"))
```

Comment:
Please, use `g_getenv` and `g_strcmp0` instead.

Reply:
Sorry I don't remember about `g_getenv`, is this some policy? Because we have a mix  of `getenv` and `g_getenv` currently. About `g_strcmp0` well we're already guaranteed here that hlsSupport is not NULL, so I'm not sure there's a point :)

Reply:
I think both are safer and more platform compatible, specially if code changes and for some reason `strcmp` is not protected by the if above.